scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/InfoView.java

```diff
@@ -117,10 +110,29 @@ public void displayItem() {
         DetailViewerRecord[] records = new DetailViewerRecord[1];
         records[0] = new DetailViewerRecord();
         this.factory.buildRecord(this.displayedItem, records[0]);
-
         this.details.setData(records);
 
+        displayExtraMembers(this.displayedItem);
+
         this.label.hide();
         this.details.show();
     }
+
+    protected Layout getLayout() {
+        /* widget that will be returned as the root layout */
+        Layout root = new Layout();
+        root.setWidth100();
+        root.setHeight100();
+
+        root.addMember(label);
+        root.addMember(details);
+
+        return root;
+    }
+
+    protected void displayExtraMembers(T object) {
```

Comment:
It's weird to have a void method called dispaly which is empty. It could be better to describe the reason in the javadoc.